internal/libpcap/inject.go

```diff
@@ -162,7 +162,7 @@ func InjectFilter(program *ebpf.ProgramSpec, filterExpr string) (err error) {
 		// stack area is safe to use. Here we use stack from -40
```

Comment:
Should I update the comment instead?
```suggestion
		// stack area is safe to use. Here we use stack from -48
```

Reply:
Let's change the offset instead.